src/runner/index.js

```diff
@@ -783,7 +783,7 @@ export default class Runner extends EventEmitter {
                 const resultOptions = {
                     ...this.configuration.getOptions(),
 
-                    dashboardUrl: reporters.find(r => r.plugin.name === 'dashboard')?.plugin.getReportUrl(),
+                    dashboardUrl: reporters.find(r => r.plugin.name === 'dashboard')?.plugin?.getReportUrl() || void 0,
```

Comment:
I guess it's better to return an empty string if the method doesn't exist, so other code won't break due to incompatible types.

```suggestion
                    dashboardUrl: reporters.find(r => r.plugin.name === 'dashboard')?.plugin?.getReportUrl() || '',
```